### PR TITLE
Add test for rating discussion replies

### PR DESCRIPTION
**Pull request by cobbal**
Also, lots of testID cleanup

refs: MBL-13691
affects: student, teacher
release note: none

### CODE REVIEW

File: CanvasCore/CanvasCore/React Native Modules/NativeUtils.m [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#import <React/RCTBridgeModule.h>
+
+@interface NativeUtils: NSObject<RCTBridgeModule>
+@end
+
+@implementation NativeUtils
+
+RCT_EXPORT_MODULE();
+
+RCT_EXPORT_METHOD(log:(NSString*)message) {
```
Comment (cobbal) [inline CanvasCore/CanvasCore/React Native Modules/NativeUtils.m]: I can never get the RN debugger working well, so I added printf debugging

**Comment by cobbal** [comment]
nightly: https://app.bitrise.io/build/445e56c9257356d7

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  You have included an invalid app. Valid values are: Student, Teacher, Parent, None
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


[MBL-13691](https://instructure.atlassian.net/browse/MBL-13691)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.3% | 93.3% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by armstrongnate** [comment]
@cobbal the nightly job you linked failed. once it passes ping me and I'll re-review.

**Pull request merged**